player/command.c

```diff
@@ -41,6 +41,7 @@
 #include "common/stats.h"
 #include "filters/f_decoder_wrapper.h"
 #include "command.h"
+#include "osdep/threads.h"
```

Comment:
I think you missed removing the pthread include at the top.

Reply:
See https://github.com/mpv-player/mpv/pull/12709#issuecomment-1793502512 for explenation why some stray `<pthread.h>` has to be there until new win32 impl.